internal/repository/repository.go

```diff
@@ -124,21 +93,8 @@ func (r *Repository) CopySources(
 ) error {
 	r.logger.Debug("copy", slog.String("origin", cloneDir))
 	for _, source := range c.Sources {
-		parts := strings.Split(
-			source.Src,
-			string(os.PathSeparator),
-		) // break up source.Src path
-		head := parts[0 : len(parts)-1] // take all path parts but last
-		tail := parts[len(parts)-1]     // take the last path part
-		cloneDirWithSrcPath := filepath.Join(
-			cloneDir,
-			strings.Join(head, string(os.PathSeparator)),
-		) // join clone dir with head
-		globbedSrc, err := glob(
-			r.appFs,
-			cloneDirWithSrcPath,
-			tail,
-		) // tail is used by glob for path matching
+		cloneDirWithSrcPath := filepath.Join(cloneDir, source.Src) // join clone dir with head
+		globbedSrc, err := afero.Glob(r.appFs, cloneDirWithSrcPath)
```

Comment:
I don't remember why the hell I didn't do this to begin with, but this is so much better, thx

Reply:
The comments would hint that afero didn't support `glob` when you wrote it.